### ISSUE TITLE
week_of_year and week_index do not exist in csv

### ISSUE THREAD

**Issue by pdabag** [jriou/2024_foph_spacetime_inla#4]
https://github.com/jriou/2024_foph_spacetime_inla/blob/9c3818980d93ec9ada8435082ff87b4e1660725e/5_time.Rmd#L29

**Comment by jriou** [comment]
It's created in `1_setup.Rmd`, lines 81 and 83. And `1_setup.Rmd` is sourced at the start of `5_time.Rmd`. At least it works for me!

**Comment by pdabag** [comment]
seems it failed to create them for me - ill check again why and report back. 
i could run all the code locally (on windows on work laptop) once i disabled renv and addapted 1_setup (see branch https://github.com/jriou/2024_foph_spacetime_inla/tree/setup-local-bag-laptop)

**Comment by pdabag** [comment]
cannot find anything in 1_setup. there is week_n (https://github.com/jriou/2024_foph_spacetime_inla/blob/9c3818980d93ec9ada8435082ff87b4e1660725e/1_setup.Rmd#L80) but no week_of_year or week_index.

**Comment by jriou** [comment]
Great that you could run eveything! Not sure about `renv`, that was @samuel-colin-foph's idea and he implemented it. Can you make a PR with your branch? I expect it will also work for the rest of the team!
Thanks Damir!

**Comment by jriou** [comment]
That's strange. My code for `1_setup.Rmd` is:

```

### Data management

```{r}
flu_season_start_week = 35
flu = flu %>% 
  mutate(week=ISOweek::ISOweek2date(paste0(temporal,"-4")), # replace ISOweek string with date of the Thursday of each week
         year=lubridate::year(week),                 
         year2=as.factor(year),                      
         week_of_year=lubridate::week(week)) %>%      
  filter(year>=2013)
flu$week_index = as.numeric(flu$week-min(flu$week))/7+1
flu$season = ifelse(flu$week_of_year<flu_season_start_week,paste0(flu$year-1,"-",flu$year),paste0(flu$year,"-",flu$year+1))
```

@samuel-colin-foph did you change this?

**Comment by pdabag** [comment]
https://github.com/jriou/2024_foph_spacetime_inla/commit/714bb0e4a56ac46189cd5d8a7e21a6c39bd676b3 seems so

**Comment by jriou** [comment]
Ok I'll merge your PR then adapt `1_setup.Rmd`.

**Comment by jriou** [comment]
Fixed in PR #6
